drivers/mpsl/clock_control/clock_control_mpsl.c

```diff
@@ -207,9 +240,45 @@ static enum clock_control_status clock_get_status(struct device *dev,
 	}
 }
 
+static int clock_async_start(struct device *dev, clock_control_subsys_t subsys,
+			     struct clock_control_async_data *data)
+{
+	struct mpsl_clock_control_data *mpsl_control_data = dev->driver_data;
```

Comment:
if LF clcok is always on then for LFLCK subsys just notify and return.

Reply:
yup, LFCLK->Clock will be handled in case 'CLOCK_CONTROL_STATUS_ON' below.